tensorflow_addons/optimizers/momentum_lars.py

```diff
@@ -0,0 +1,206 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Layer-wise Adaptive Rate Scaling optimizer for large-batch training."""
+
+import tensorflow.compat.v2 as tf
+
+from tensorflow.python.ops import array_ops
+from tensorflow.python.ops import math_ops
+from tensorflow.python.training import training_ops
```

Comment:
please use
```
import tensorflow as tf
```
and access API like
```
tf.where
tf.cast
```
e.t.c..

However, since `training_ops` has no other way to access, you can use
```python
# TODO: Find public API alternatives to these
from tensorflow.python.training import training_ops
```